shorebird_code_push/lib/src/generated/updater_bindings.g.dart

```diff
@@ -2084,20 +2084,20 @@ class UpdaterBindings {
   late final _shorebird_init = _shorebird_initPtr.asFunction<
       bool Function(ffi.Pointer<AppParameters>, ffi.Pointer<ffi.Char>)>();
 
-  /// Return the active patch number, or NULL if there is no active patch.
-  ffi.Pointer<ffi.Char> shorebird_next_boot_patch_number() {
+  /// The patch number that will boot on the next run of the app, or 0 if there is
+  /// no next patch.
+  int shorebird_next_boot_patch_number() {
     return _shorebird_next_boot_patch_number();
   }
 
   late final _shorebird_next_boot_patch_numberPtr =
-      _lookup<ffi.NativeFunction<ffi.Pointer<ffi.Char> Function()>>(
+      _lookup<ffi.NativeFunction<ffi.UintPtr Function()>>(
```

Comment:
FWIW, We will end up needing to handle both partial and full failure of lookups in this code.  partial will happen when there is version skew between this package and the version of shorebird in use (wrong verison pulled from pub) and full failure will happen every time the user uses `flutter` directly instead of `shorebird`.